packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -333,6 +343,16 @@ function build(
         // eslint-disable-next-line no-use-before-define
         return queueMessage(vpid, prop, args, returnedP);
       },
+      get(p, prop) {
+        // Support: p~.[prop]
+        lsdebug(`makePipelinablePromise handler.get (${vpid})`);
+        if (!handlerActive) {
+          console.error(`mIPromise handler called after resolution`);
+          assert.fail(X`mIPromise handler called after resolution`);
+        }
+        // FIXME: Actually pipeline.
```

Comment:
Let’s track this.

Reply:
I'm leaving it for @warner to clean up once he implements the `*SendOnly` variants in liveSlots.

If he wants an additional issue just for this, I'm fine with that, but I don't want to make one.

Reply:
I was being lazy.  Now we have #3894.